athena-jdbc/src/main/java/com/amazonaws/athena/connectors/jdbc/manager/JdbcRecordHandler.java

```diff
@@ -163,7 +163,10 @@ public void readWithConstraint(BlockSpiller blockSpiller, ReadRecordsRequest rea
                 }
                 LOGGER.info("{} rows returned by database.", rowsReturnedFromDatabase);
 
-                connection.commit();
+                // Not performing commit() in case of Azure serverless environment
```

Comment:
Can this comment expand further as to why we should not commit if we are in an Azure serverless environment?

Reply:
Taken care.